packages/server/src/script/fetchMGErrors.ts

```diff
@@ -0,0 +1,131 @@
+import { Logging } from '@google-cloud/logging'
+import { processEnvOrThrow } from '../common'
+import { MailgunHookBody, MailgunEventReason } from '../service/webhooks'
+import { promises as fs } from 'fs'
+
+interface EntryResponse {
+  data: MailgunHookBody
+}
+
+/**
+ * * The first is the starting date (inclusive) of the script;
```

Comment:
This is me nit picking, and I'm totally cool with this documentation, but I would suggest following this pattern in the future:

```
/**
* Description of what the function is doing.
* @param: {name1}: {type1}{description}
* @param: {name2}: {type2}{description}
* @returns {type}{description}
*/ 
```

Doesn't have to be exactly like this, but you get the idea.